src/MoBi.Presentation/Presenter/EditEventGroupBuildingBlockPresenter.cs

```diff
@@ -130,9 +130,9 @@ private IEnumerable<T> allPresenterImplementing<T>()
          return AllSubPresenters.OfType<T>();
       }
 
-      protected override (bool canHandle, IContainer parentObject) SpecificCanHandle(IObjectBase selectedObject)
+      protected override (bool canHandle, IContainer containerObject) SpecificCanHandle(IObjectBase selectedObject)
       {
-         return (shouldShow(selectedObject), null);
+         return (shouldShow(selectedObject), selectedObject as IContainer);
```

Comment:
That's the real change. In v11.2 the method was:

```
      protected override Tuple<bool, IObjectBase> SpecificCanHandle(IObjectBase selectedObject)
      {
         return new Tuple<bool, IObjectBase>(shouldShow(selectedObject), selectedObject);
      }
```
So rather than naming this parentObject, I changed it to containerObject. In this case, the selected object is a Container.